project/Build.scala

```diff
@@ -13,14 +13,11 @@ import scoverage.ScoverageSbtPlugin
 
 object Scrooge extends Build {
   val branch = Process("git" :: "rev-parse" :: "--abbrev-ref" :: "HEAD" :: Nil).!!.trim
-  val suffix = if (branch == "master") "" else "-SNAPSHOT"
```

Comment:
please don't turn this off, it's extremely useful for making changes in libs that depend on other libs that haven't been published yet.